components/admin-panel/sections/accounting/CreateAccountingCategoryModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { AccountingCategory, AccountingCategoryKind } from '../../../../lib/graphql/types/v2/graphql';
+
+import StyledButton from '../../../StyledButton';
+import StyledModal, { ModalBody, ModalFooter, ModalHeader } from '../../../StyledModal';
+
+import {
+  AccountingCategoryForm,
+  AccountingCategoryKindI18n,
+  useAccoutingCategoryFormik,
+} from './AccountingCategoryForm';
+
+type CreateAccountingCategoryModalProps = {
+  onClose: () => void;
+  onCreate: (category: Pick<AccountingCategory, 'name' | 'friendlyName' | 'code' | 'expensesTypes'>) => Promise<void>;
+};
+
+export function CreateAccountingCategoryModal(props: CreateAccountingCategoryModalProps) {
+  const intl = useIntl();
+  const { onCreate } = props;
+  const createFundCategoryPicker = React.useCallback(
```

Comment:
Not sure why "Fund", typo I suppose?

```suggestion
  const createAccountingCategoryPicker = React.useCallback(
```